spark/v3.5/spark-runtime/LICENSE

```diff
@@ -379,14 +334,6 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains code from Presto.
```

Comment:
I don't find presto in the jar (on mention of Presto copied code).

Reply:
@findepi Do you know why this is here? I can't find any references in the Spark repository to presto either (not in their notice file)

Reply:
https://github.com/apache/iceberg/blob/main/spark/v3.5/spark-extensions/src/main/antlr/org.apache.spark.sql.catalyst.parser.extensions/IcebergSqlExtensions.g4#L19

So it seems like our Spark SQL extensions defined in the ANTLR grammar are an adaptation of OSS Spark and Presto grammars. I think we may want to keep this then.

Reply:
@RussellSpitzer no, i was not involved in this

Reply:
@amogh-jahagirdar if it's true, it should be documented in Iceberg LICENSE.

Reply:
It's not a binary inclusion it's code copy. So not the same form.

Reply:
@amogh-jahagirdar good catch, I see this now in the root `LICENSE`. Note for later: we should provide the full path of the code using/copying code to quickly identify the module where it applies.

Reply:
The runtime bundles should contain nearly all of the same LICENSE content as the source tarball because the source is almost entirely present (at least, core, api, common, parquet, data, etc are in every bundle). Things that are in the root license file should be preserved unless they are known to be specific to a runtime bundle. Similarly, when adding to the root license we need to update all of the bundle license files.

Reply:
My point is that the root `LICENSE` should contain the location of the code copy. In this case, Presto has been copied to Spark, so it applies to source distribution and spark bundle jar, not flink for instance.